pkg/cloud/services/eks/addons.go

```diff
@@ -196,7 +196,7 @@ func (s *Service) translateAPIToAddon(addons []ekscontrolplanev1.Addon) []*eksad
 		convertedAddon := &eksaddons.EKSAddon{
 			Name:                  &addon.Name,
 			Version:               &addon.Version,
-			Tags:                  ngTags(s.scope.Cluster.ClusterName, s.scope.AdditionalTags()),
+			Tags:                  ngTags(s.scope.Cluster.Name, s.scope.AdditionalTags()),
```

Comment:
I feel like this name should be the Kubernetes cluster name instead of CAPI cluster name, since they are different in EKS.
Not sure though.

Reply:
You mean EKS cluster name? I don't know why the original code is using CAPI cluster name but that is a great point. 
Let's ask @richardcase.

Reply:
I'd be tempted to use `s.scope.Cluster.Name` and not the eks cluster name. An addon is always a child of a EKS cluster (and so you have the name).